src/databases/VTK/avtVTKWriter.C

```diff
@@ -258,39 +263,41 @@ avtVTKWriter::WriteChunk(vtkDataSet *ds, int chunk)
         mn->Delete();
     }
 
-    // Write any non-operator, non-auto expressions
-    if (exprList.GetNumExpressions())
-    {
-        vtkStringArray *mn = vtkStringArray::New();
-        mn->SetNumberOfValues(exprList.GetNumExpressions());
-        int used = 0;
-        for (int i = 0; i < exprList.GetNumExpressions(); i++)
-        {
-            Expression const expr = exprList.GetExpressions(i);
-
-            if (expr.GetFromOperator()) continue;
-            if (expr.GetAutoExpression()) continue;
-
-            string vtypestr = "unknown";
-            switch (expr.GetType())
-            {
-                case Expression::CurveMeshVar:  vtypestr = "curve";    break;
-                case Expression::ScalarMeshVar: vtypestr = "scalar";   break;
-                case Expression::VectorMeshVar: vtypestr = "vector";   break;
-                case Expression::TensorMeshVar: vtypestr = "tensor";   break;
-                case Expression::ArrayMeshVar:  vtypestr = "array";    break;
-                case Expression::Material:      vtypestr = "material"; break;
-                case Expression::Species:       vtypestr = "species";  break;
-                default: break;
-            }
-
-            mn->SetValue(used++, expr.GetName() + ";" + vtypestr + ";" + expr.GetDefinition());
-        }
-        mn->SetNumberOfValues(used);
-        mn->SetName("VisItExpressions");
-        ds->GetFieldData()->AddArray(mn);
-        mn->Delete();
-    }
+    // The team decided that expressions should not be output in this way; however,
```

Comment:
FWIW...I think the right thing to do is remove code entirely.

Also, in cases where you do maybe want to keep some code around but exclude it from compilation, I always find it easier to use something like....

```

#if 0
.
.
.
#endif
```

Or, maybe...

```

#ifndef ENABLE_FOOBAR_BLOCK

// lines of code for FOOBAR operation.

#endif
```

where the latter approach does allow one to re-enable that code by doing something like `CPPFLAGS="-DENABLE_FOOBAR_BLOCK"`

Reply:
Thanks for this! I am still learning so I appreciate all the feedback very much 😄

Reply:
Wait 'til you try to do something similar in Python 😉 . Nothing like `#if 0...#endif` there.